app/setup.py

```diff
@@ -66,6 +68,8 @@ def create_app(setting_overrides=None):  # noqa: C901  pylint: disable=too-compl
     application = Flask(__name__, static_url_path='/s', static_folder='../static')
     application.config.from_object(settings)
 
+    setup_secure_headers(application)
```

Comment:
We should move this further down the setup. Especially after line 87 as that could change the behaviour.
I would probably put it after line 90 when we log out the version.